db/migrate/20170215211450_create_comment.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.belongs_to :death_record
       t.belongs_to :user
       t.string :content
+      t.boolean :requested_edits, default: false
```

Comment:
Overall this seems right; could we tweak it so we add this column in a new migration, rather than just updating the existing one? We've got the demo server up, and ideally I'd just be able to run the migration on that server rather than bringing the whole thing down and up again (and recreating all the user accounts, etc).